Re-Organize OpenMusic folders

karst opened the pull request:
This PR creates a new folder called replacements which will take the place of the new styles.

It also includes two TXT files and a preset object.json

OpenRCT2 music names.txt is to be further discussed upon.

Rendering guide.txt is the standard format we need to work with.

karst committed:
Add files via upload

A New Dodgem theme.

karst committed:
Add files via upload

A New Dodgem theme.

karst committed:
Revert "Add files via upload"

This reverts commit a10815e0487114f0912c915431315d2428ff45ab.

karst committed:
Revert "Add files via upload"

This reverts commit a10815e0487114f0912c915431315d2428ff45ab.

karst committed:
add files for new Dodgem theme

WAV & json file

karst committed:
add files for new Dodgem theme

WAV & json file

karst committed:
replaced WAV for FLAC

karst committed:
replaced WAV for FLAC

karst committed:
File name changes

karst committed:
File name changes

karst committed:
Real name it is :)

karst committed:
Real name it is :)

karst committed:
Added sound effect for news & balloon pop

These are not necessarily final.

karst committed:
OpenMusic reorg

Created a new folder with replacements. Moved the fairground organ & title theme into here.

Added two txt files.
OpenRCT2 music names.txt is a document with explaination and ideas for name changes (to be discussed along the way).
Rendering guide.txt shows the

karst committed:
Removed SoundFX

Accidental transfer

karst committed:
Moved files to replacements folder.

karst committed:
Moved files to replacements folder.

karst committed:
Formatting

karst committed:
Formatting

oli414 commented:
Awesome work. It might be worth moving the TXT files to the Wiki. Or turning them into MDs to make them a bit more pleasant to read.

ExileFox commented:
MD is probably preferred.

karst committed:
Updated standard

karst committed:
Updated standard

karst committed:
Update Rendering guide.txt

Broxzier commented on replacements/Rendering guide.txt:
```diff
@@ -0,0 +1,24 @@
+-----------------------
+For finished works:
+
+- A FLAC file of the finished product. This can be included in the official soundtrack released beside the game. Put thisone in \replacements\soundtrack\
```
Typo:
```suggestion
- A FLAC file of the finished product. This can be included in the official soundtrack released beside the game. Put this in \replacements\soundtrack\
```

karst committed:
Update replacements/Rendering guide.txt

Co-authored-by: Hielke Morsink <hielke.morsink@gmail.com>

karst committed:
Update format

karst committed:
Update format

Gymnasiast committed:
Make additional changes

Gymnasiast committed:
Make additional changes

karst committed:
Update to the new discussed format

karst committed:
Create README.md

I have made a `README.md` which includes both  `OpenRCT2 music names.txt`  and `Rendering guide.txt` which makes those redundant.

We can include more info on here on the project if necessary. But I wouldn't know what to add.

karst committed:
Add confirmation for merge to README.md

karst committed:
Formatting Fix

Some enters were missing making some lines hard to read.

karst commented:
I have made a README.md, which makes the included txt files redundant, what do you guys prefer?

Changes can always be made in a later point in time, this is just to lay down a base.

You can preview the README.md over here: https://github.com/karst/OpenMusic/blob/openmusic-reorg/replacements/README.md

karst committed:
Fix accidental reversion

karst committed:
Removed redundant files

Merged into README.md

karst committed:
Why different names, you might ask?

I have added a question to the top of the page to explain why I've decided it's smart to have different names from the RCT2 names.

karst committed:
Grammar.....

karst committed:
Fairground update

I have added another readme for the plans of the Fairground organ.
I have also updated the README.md to include more intentions (asset packs)
And I have organized the progress list by cover > alphabetical. (With exception of the title screen which is now on top)

karst committed:
openrct2.music.relocate

Relocated and renamed the folders of medieval, ragtime and the title theme to include openrct2.music in front of them

karst committed:
Update README.md

karst committed:
Merge branch 'OpenRCT2:master' into openmusic-reorg

karst committed:
Add object collumn

karst committed:
Update README.md

✔ Acid style

karst committed:
Update to new standard.

karst committed:
Merge branch 'master' into openmusic-reorg

karst committed:
Merge branch 'master' into openmusic-reorg

karst committed:
Update json files to flac instead of wav

IntelOrca commented:
@karst can you fix the CI? Only include object.json files for tracks that will compile successfully.

karst commented:
Yea for sure. Do you want me to remove the object.json file from the replacement's root aswell?
I could move the code to the README.md or remove it entirely so people can look at already included folders

IntelOrca commented:
@karst that one might be okay, see if CI fails, if it does, move it or remove it. Call it `template.object.json.`

karst committed:
Remove object.json from WIPs and root

karst committed:
Update README.md

Template was the word i was looking for :P

karst committed:
Update README.md

karst commented:
CI is exiting with the following error:

```
ENOENT: no such file or directory, open 'replacements/openrct2.music.fairground/object.json'
Error: Process completed with exit code 1.
```

It's erroring out because it cannot find an `object.json` file. What do I do to negate this?

IntelOrca commented:
I can fix that

IntelOrca commented:
I can't get it to rebuild against lastest version of master. If you rebase or commit another thing it should refresh.

karst committed:
Update README.md

karst commented:
Perfect! That did it :)

IntelOrca commented:
@Gymnasiast are you still requesting a review?

karst committed:
Add discussion page for Fairground style

karst committed:
Update README.md

tupaschoal commented:
Ping @Gymnasiast

Gymnasiast commented:
I will do a review soon, thanks for reminding me!

karst committed:
Merge branch 'OpenRCT2:master' into openmusic-reorg

karst committed:
FLAC/OGG support in the Readme

karst committed:
Merge branch 'openmusic-reorg' of https://github.com/karst/OpenMusic into openmusic-reorg

karst commented:
Bump

Gymnasiast commented on replacements/openrct2.music.fairground/README.md:
```diff
@@ -0,0 +1,30 @@
+# OpenRCT2/OpenMusic/replacements/openrct2.music.fairground
+
+The following project is to replace the RCT2 fairground organ with both remastered aswell as new songs to the merry-go-round.  
```
```suggestion
The following project is to replace the RCT2 fairground organ with both remastered as well as new songs to the merry-go-round.  
```

Gymnasiast commented on replacements/README.md:
```diff
@@ -0,0 +1,116 @@
+ There will be three asset packs:
+ 
+ * `rct2.music` - All the music from RCT2.
+ * `openrct2.music.cover` - Cover of non-copyright music, e.g. merry-go-round, pirate theme, ragtime, organ etc.
```
What would be in the places of copyrighted music?

karst replied:
openrct2.music.alternative would replace the copyrighted music aswell as be available for everyone to use as extra tracks if they have RCT2 installed.

IntelOrca replied:
Currently it replaces them with our OpenRCT2 equivalent. For example, space style will play galaxy style. Techno style will play Acid Style. Dodgems will play `openrct2.music.dodgems`

karst committed:
Update replacements/openrct2.music.fairground/README.md

Co-authored-by: Michael Steenbeek <m.o.steenbeek@gmail.com>

Gymnasiast reviewed (approved):
I don’t know if this is necessarily the final organisation or if we want to change things around, but it’s probably better than what we have anyway.

karst committed:
Update README.md

I have added a few more lines of details that I have forgotten and things that are more finished rather than sounding like a guess. I have also included a link to the creative commons website for the licensing of our music.
